src/Features/CSharp/Portable/CodeRefactorings/UseRecursivePatterns/UseRecursivePatternsCodeRefactoringProvider.cs

```diff
@@ -77,6 +81,17 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 _ => null
             };
 
+        private static bool IsFixableNode(SyntaxNode node)
+            => node switch
+            {
+                BinaryExpressionSyntax(LogicalAndExpression) => true,
+                CasePatternSwitchLabelSyntax { WhenClause: { } whenClause } => true,
+                SwitchExpressionArmSyntax { WhenClause: { } whenClause } => true,
+                WhenClauseSyntax { Parent: CasePatternSwitchLabelSyntax } => true,
+                WhenClauseSyntax { Parent: SwitchExpressionArmSyntax } => true,
+                _ => false
+            };
+
```

Comment:
```suggestion
        private static bool IsFixableNode(SyntaxNode node)
            => node switch
            {
                BinaryExpressionSyntax(LogicalAndExpression) => true,
                CasePatternSwitchLabelSyntax { WhenClause: not null } => true,
                SwitchExpressionArmSyntax { WhenClause: not null } => true,
                WhenClauseSyntax { Parent: CasePatternSwitchLabelSyntax } => true,
                WhenClauseSyntax { Parent: SwitchExpressionArmSyntax } => true,
                _ => false
            };

```

Reply:
I'll fold this suggestion into the next PR and avoid re-running tests on this PR.